internal/overlord/state/state.go

```diff
@@ -316,12 +315,18 @@ func (s *State) VerifyReboot(curBootID string) error {
 	return nil
 }
 
+// BootID returns the current boot id set by VerifyReboot.
+func (s *State) BootID() string {
```

Comment:
We'll need to move this as well at some point, but doing the right thing should be obvious once the API is fixed as described above.